BeVolt/App/src/Voltage.c

```diff
@@ -60,21 +60,29 @@ ErrorStatus Voltage_UpdateMeasurements(void){
 
 /** Voltage_IsSafe
  * Checks if all modules are safe
- * @return SAFE or FAIL
+ * @return SAFE or danger: UNDERVOLTAGE or OVERVOLTAGE
  */
 SafetyStatus Voltage_IsSafe(void){
 	for(int32_t i = 0; i < NUM_BATTERY_MODULES; i++){
 		uint16_t voltage = Modules[i / 12].cells.c_codes[i % 12];
-		if(voltage > MAX_VOLTAGE_LIMIT || voltage < MIN_VOLTAGE_LIMIT){
-			return DANGER;
+		
+		// VOLTAGE_LIMITS are in floating point. The LTC6811 sends the voltage data
+		// as unsigned 16-bit fixed point integers with a resolution of 0.00001
+		if(voltage > MAX_VOLTAGE_LIMIT * 10000){
```

Comment:
That's an odd (even) number of zeros, and I would feel better if they were given their own definition (maybe `#define LTC6811_SCALING_FACTOR 10000` or something) so that we can eliminate the likelihood of mistakes.

Reply:
That's a good idea